### DISCUSSION TITLE
No `size` for components

### DISCUSSION THREAD

**Discussion by slowfound** [zeit-ui/react#586] [Q&A] [answered]
In my app, the Buttons are not getting influenced by the ``size`` prop at all. I'm fairly certain that I'm not overriding the button height anywhere in my styling.

**Comment by ofekashery** [comment, marked as answer]
Are you using the RC version of Geist? If so, please refer to the RC documentation: [react-rc.geist-ui.dev](https://react-rc.geist-ui.dev).

----

Geist no longer uses the ``size`` prop to control component sizing, you can now dynamically scale components with the ``scale`` prop.

[**💡 Learn more in the Scaleable documentation**](https://react-rc.geist-ui.dev/en-us/guide/scaleable)

**Reply by slowfound** [reply]
Yes, I checked my `package.json`  file and I was using the RC version, I tried the `scale={x}` prop and it works. Thank you so much.
